src/apps/common/submodules/mediaSelect/mediaSelect.js

```diff
@@ -20,6 +20,11 @@ define(function(require) {
 		 * @param {Array} args.options
 		 * @param {Array} args.mimeTypes - Allowed file formats to be uploaded if `dragableUpload` is true
 		 * @param {Boolean} [args.dragableUpload]
+		 * @param {Object} [args.tts] - Enable text to speech tab if present
+		 * @param {String} [args.tts.id] - Current media id, used when the media is updated
+		 * @param {String} [args.tts.text] - Current tts text
+		 * @param {String} [args.tts.name]
+		 * @param {String} [args.tts.type]
```

Comment:
Although not documented (😞) it looks like `args` already has a prop that contains the selected option (`selectedOption`), which we could use to hold the media object.

Then `tts` would be a simple boolean (could be renamed to `withTts` or something alike to denote the boolean value) to determine if the tab should be shown.